samples/cpp/object_detection_sample_ssd/main.cpp

```diff
@@ -86,7 +89,7 @@ int main(int argc, char* argv[]) {
         // ------------------------------ Get Available Devices
         // ------------------------------------------------------
```

Comment:
Can we make such huge comments shorter? Like here https://github.com/openvinotoolkit/openvino/blob/master/samples/cpp/hello_classification/main.cpp#L39 in other samples which were not rewritten by me - the same

Reply:
actually, I'm going to remove this and style transfer sample, but for now, ok, reduced comments a bit